src/treewalk.coffee

```diff
@@ -112,6 +118,7 @@ exports.createTreewalkParser = (parse, config, root) ->
               color: @getColor node, rules
               classes: padRules(wrapRules ? rules).concat(@getShape(node, rules))
               parseContext: rules[0] #(if wrap? then wrap.type else rules[0])
+              buttons: config.BUTTON_CALLBACK(@opts, node) ? null
```

Comment:
This breaks C mode right now. Can you either add an empty `BUTTON_CALLBACK` to C mode, or make `BUTTON_CALLBACK` optional by checking to see whether it exists before calling it?